Readme.md

```diff
@@ -1,9 +0,0 @@
-This project is the exercises I did as
```

Comment:
Looks like you deleted this, and we don't want to.  Remind me and I'll show you how to fix in Friday's class.